docker/docker_container_wrapper.go

```diff
@@ -22,6 +22,14 @@ func (c *dockerContainerWrapper) getExposedTCPPorts() (ports []int) {
 	return
 }
 
+func (c *dockerContainerWrapper) getTags() []string {
+	if tags, exist := c.getEnv()["SRV_TAGS"]; exist {
+		tagsArray := strings.Split(tags, ",")
```

Comment:
@alaa do you know if there are any constraints for tag value? I haven't found any details about it in consul documentation